parts/k8s/cloud-init/artifacts/cse_customcloud.sh

```diff
@@ -78,6 +87,44 @@ configureK8sCustomCloud() {
   ifconfig eth0 mtu 1350
 
   set -x
+  {{else}}
+  ensureCustomCloudRootCertificates
```

Comment:
If these two funcs aren’t called in an Azure Stack context should we add an if not AzureStackCloud block around them so that Azure Stack cloud-int contexts don’t have the superflous bytes?